test/specs/mainnet/swap/Smoke_CrossChainQuote.spec.js

```diff
@@ -0,0 +1,4195 @@
+      "0xa5494Ed2eB09F37b4b0526a8e4789565c226C84f",
+      "The EOA Address is not displayed correctly."
+    );
+
+    // Compute the smart wallet address
+    let smartWalletOutput = await sdkMainNet.computeContractAccount();
+    smartWalletAddress = smartWalletOutput.address;
+    console.log("Smart wallet address: ", smartWalletAddress);
+
+    assert.strictEqual(
+      smartWalletAddress,
+      "0x666E17ad27fB620D7519477f3b33d809775d65Fe",
+      "The smart address is not displayed correctly."
+    );
+
+    let ArbitrumUSDC = "0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8"; // Arbitrum - USDC
+    let MaticUSDC = "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174"; // Matic - USDC
+
+    let fromChainId = NETWORK_NAME_TO_CHAIN_ID[NetworkNames.Arbitrum];
+    let toChainId = NETWORK_NAME_TO_CHAIN_ID[NetworkNames.Matic];
+    let fromTokenAddress = ArbitrumUSDC;
+    let toTokenAddress = MaticUSDC;
+    let fromAmount = ethers.utils.parseUnits("0.5", 6);
+
+    let quoteRequestPayload = {
+      fromChainId: fromChainId,
+      toChainId: toChainId,
+      fromTokenAddress: fromTokenAddress,
+      toTokenAddress: toTokenAddress,
+      fromAmount: fromAmount,
+      serviceProvider: CrossChainServiceProvider.SocketV2, // Optional parameter
+    };
+    // console.log(quoteRequestPayload);
+
+    try {
+      assert.isNumber(
+        quoteRequestPayload.fromChainId,
+        "The fromChainId value is not displayed in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNumber(
+        quoteRequestPayload.toChainId,
+        "The toChainId value is not displayed in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        quoteRequestPayload.fromTokenAddress,
+        "0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8",
+        "The fromTokenAddress value is not displayed correct in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        quoteRequestPayload.toTokenAddress,
+        "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
+        "The toTokenAddress value is not displayed correct in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        quoteRequestPayload.fromAmount,
+        "The fromAmount value is not displayed in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        quoteRequestPayload.serviceProvider,
+        "SocketV2",
+        "The serviceProvider value is not displayed correct in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    // GET THE CROSS CHAIN QUOTES
+    let batchCrossChainTransaction;
+    let quotes = await sdkMainNet.getCrossChainQuotes(quoteRequestPayload);
+    // console.log("Quotes::: ", quotes);
```

Comment:
Please remember to remove any dead code, or leave a comment into why it's here.

Reply:
Removed the commented code from the test cases.